app/customers.py

```diff
@@ -0,0 +1,118 @@
+import json
+import math
+
+from app.shop import Shop
+
+
+class Customer:
+
+    def __init__(
+            self,
+            name: str,
+            product_cart: dict,
+            location: list[int],
+            money: int,
+            car: dict
+    ) -> None:
+        self.name = name
+        self.product_cart = product_cart
+        self.location = location
+        self.money = money
+        self.car = car
+
+    with open("app/config.json", "r") as file:
+        data = json.load(file)
```

Comment:
Reading the JSON file once and storing its data for subsequent use is a recommended practice